<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameTypeConfigOld</class>
 <widget class="QWidget" name="FrameTypeConfigOld">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>580</width>
     <height>550</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QRadioButton" name="plusRadioButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>'Plus' Style</string>
      </property>
      <property name="iconSize">
       <size>
        <width>500</width>
        <height>150</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="styleSheet">
       <string notr="true">QPushButton {
    /*min-height: 16px;*/
	border: 0px;
    margin: 3px;
	border-radius: 2px;
	padding-top: 4px;
	padding-left: 8px;
	padding-right: 8px;
	padding-bottom: 4px;
	background-color: 0
}

QPushButton:pressed {
	border-width: 1px dashed #F0F0F0
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../../qgroundcontrol.qrc">
        <normaloff>:/files/images/mavs/frames_plus.png</normaloff>:/files/images/mavs/frames_plus.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>400</width>
        <height>80</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QRadioButton" name="xRadioButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>'X' Style</string>
      </property>
      <property name="iconSize">
       <size>
        <width>500</width>
        <height>150</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="styleSheet">
       <string notr="true">QPushButton {
    /*min-height: 16px;*/
	border: 0px;
    margin: 3px;
	border-radius: 2px;
	padding-top: 4px;
	padding-left: 8px;
	padding-right: 8px;
	padding-bottom: 4px;
	background-color: 0
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../../qgroundcontrol.qrc">
        <normaloff>:/files/images/mavs/frames_x.png</normaloff>:/files/images/mavs/frames_x.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>380</width>
        <height>80</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QRadioButton" name="vRadioButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>'V' Style</string>
      </property>
      <property name="iconSize">
       <size>
        <width>500</width>
        <height>120</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="1" rowspan="2">
     <widget class="QPushButton" name="pushButton_5">
      <property name="styleSheet">
       <string notr="true">QPushButton {
    /*min-height: 16px;*/
	border: 0px;
    margin: 3px;
	border-radius: 2px;
	padding-top: 4px;
	padding-left: 8px;
	padding-right: 8px;
	padding-bottom: 4px;
	background-color: 0
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../../qgroundcontrol.qrc">
        <normaloff>:/files/images/firmware/quady.png</normaloff>:/files/images/firmware/quady.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0">
     <widget class="QRadioButton" name="hRadioButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>'H' Style</string>
      </property>
      <property name="iconSize">
       <size>
        <width>500</width>
        <height>120</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    /*min-height: 16px;*/
	border: 0px;
    margin: 3px;
	border-radius: 2px;
	padding-top: 4px;
	padding-left: 8px;
	padding-right: 8px;
	padding-bottom: 4px;
	background-color: 0
}
</string>
        </property>
        <property name="text">
         <string>`</string>
        </property>
        <property name="icon">
         <iconset resource="../../../qgroundcontrol.qrc">
          <normaloff>:/files/images/mavs/frames_quad_I.png</normaloff>:/files/images/mavs/frames_quad_I.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>NOTE: X and H are not interchangable as the prop rotation changes</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <widget class="QRadioButton" name="newY6radioButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>Y6B</string>
      </property>
      <property name="iconSize">
       <size>
        <width>500</width>
        <height>120</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    /*min-height: 16px;*/
	border: 0px;
    margin: 3px;
	border-radius: 2px;
	padding-top: 4px;
	padding-left: 8px;
	padding-right: 8px;
	padding-bottom: 4px;
	background-color: 0
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../qgroundcontrol.qrc">
          <normaloff>:/files/images/mavs/frames-Y6B.png</normaloff>:/files/images/mavs/frames-Y6B.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>NOTE: This is the new Y6 configuration with all props on the bottom rotating CCW and on top CW </string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../../qgroundcontrol.qrc"/>
 </resources>
 <connections/>
</ui>
